lib/awful/tag.lua

```diff
@@ -1370,11 +1366,6 @@ object.properties(capi.tag, {
     setter_fallback = tag.setproperty,
 })
 
--- fix a load loop
-client = require("awful.client")
-capi.client.connect_signal("manage", function(c) client.object.to_selected_tags(c) end)
```

Comment:
I fear that this has to stay. Perhaps in a `gears.timer.delayed_call`.
For some reason, using `awful.rules` is still optional (`require("awful")` does not load `awful.rules`). People who don't use `awful.rules` get their new clients tagged due to this.